src/libdrakvuf/libdrakvuf.h

```diff
@@ -223,6 +223,34 @@ struct drakvuf_trap {
     void *data;
 };
 
+
+////////////////////////////////////////////////////////////////////////////
+
+// IMHO these definitions must be placed within another file, named
+// libdrakvuf-windows.h or something similar
+
+// For get_previous_mode...
+typedef enum _MODE {
```

Comment:
@tklengyel : Agreed

So I infere you agree leaving these definitions on libdrakvuf.h... right? :)

On the other hand, as the comments on the file say, I wonder if placing definitions being used from the plugins on different files for different platforms (libdrakvuf-windows.h, libdrakvuf-linux.h etc.) makes sense for you.